test/liquid_handle_unit_test.py

```diff
@@ -124,6 +124,7 @@ class LiquidHandleMethodTester(object):
     )
     asp_transport = LiquidHandle.builders.transport(
         volume=Unit(1, "uL"),
+        density=None,
```

Comment:
We shouldn't be changing old tests to meet new specs, this means this isn't backwards compatible. My comment above about moving the density argument to the end of the function argument list should fix this (and you should then be able to remove `density=None` to get this test to pass again)

Reply:
removed. passes fine.